userspace/engine/stats_manager.h

```diff
@@ -63,7 +63,6 @@ class stats_manager
 		const indexed_vector<falco_rule>& rules,
 		std::string& out) const;
 
-private:
```

Comment:
I think this is the wrong move. Currently, this class is the sole manager of these values by providing a single entry point to update them. Making them all public means that any part of the code which has access to the stats_manager can change them which may have unpredictable side effects.
Having a dedicated read API would keep the responsibility within this class.

Reply:
I almost anticipated that comment from someone 😆 sure I expand the API instead.